### DISCUSSION TITLE
Best way to run bilingual Quarto website/Blog

### DISCUSSION THREAD

**Discussion by BatoolMM** [quarto-dev/quarto-cli#2226] [Q&A]
I am trying to run a bilingual blog with Quarto but not sure what is the best way to do that. Am I supposed to have 2 YAML, one for each language but how to link them?

I found this [book](https://www.patolojiatlasi.com/EN/), which seems that used quarto by @sbalci but not sure what approach he is taking. Any clue on how the config file should be construed, would be appreciated. I'll be using RTL and LTR languages in [this quarto Blog](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA-en-blog)?

**Comment by sbalci** [comment]
Please see explanation here:
https://github.com/quarto-dev/quarto-cli/issues/275#issuecomment-1073259436
and github action here: https://github.com/patolojiatlasi/patolojiatlasi.github.io/blob/main/.github/workflows/Quarto-Render-Bilingual-Book-Push-Tweet-Updates.yml
